### PR TITLE
v3.11.4 Release (Deprecated PR)

### PR DESCRIPTION
**Pull request by canyon289**
Staging a release to rectify my v3.11.3 issue on pypi given that we cant rerelease a version on pypi

### CODE REVIEW

File: RELEASE-NOTES.md
Comment (mjhajharia) [inline RELEASE-NOTES.md:28]: i don't know how releases work, just curious if this line should be there if we're doing 3.11.4 and skipping 3.11..3

**Review by mjhajharia — DISMISSED** [review]

**Comment by Sayam753** [comment]
`v3.11.4` docs will look majorly like - https://pymc3.readthedocs.io/en/v3/
Things that hit my mind just now - 
- The sponsorship links got changed and they are broken on v3 landing page even after recompiling docs. Shall we remove the broken links from [index.rst](https://github.com/pymc-devs/pymc3/blob/v3/docs/source/index.rst)?
- Even the links at Books + Videos page are broken - https://pymc3.readthedocs.io/en/v3/learn.html. What to do with them?

I should have thought about this earlier 😞 

cc @twiecki

**Comment by canyon289** [comment]
> `v3.11.4` docs will look majorly like - https://pymc3.readthedocs.io/en/v3/
> Things that hit my mind just now -
> 
>     * The sponsorship links got changed and they are broken on v3 landing page even after recompiling docs. Shall we remove the broken links from [index.rst](https://github.com/pymc-devs/pymc3/blob/v3/docs/source/index.rst)?
> 
>     * Even the links at Books + Videos page are broken - https://pymc3.readthedocs.io/en/v3/learn.html. What to do with them?
> 
> 
> I should have thought about this earlier 😞
> 
> cc @twiecki

Dont worry, we can fix them now that were doing another release :)

File: RELEASE-NOTES.md
Comment (canyon289) [reply RELEASE-NOTES.md:28]: ITs a good question. We technically did release 3.11.3 so if someone installs from github this is the correct changelog (now that I fixed the tags)

Since we have some docfixes to do below I think thatll end up being the extent of 3.11.4s changes

**Comment by Sayam753** [comment]
Awesome. I'll roll out a PR for doc fixes in next 12 hours.

**Review by michaelosthege — DISMISSED** [review]

File: RELEASE-NOTES.md
Comment (twiecki) [reply RELEASE-NOTES.md:28]: I agree, I think we should just mark 3.11.3 as broken and move release notes up.

**Comment by canyon289** [comment]
This is just a question about booking keeping. Should we merge anything other than the release notes and a version bump in a release pr? The code will end up being the same just curious about the git history type stuff

My opinion is no, if anyone else agrees Ill split up this PR.

Edit:
Thinking about this I realized the opinions from at least Sayam or Thomas must be yes its fine given the merge so I'll just merge this release branch as is once given approval

File: RELEASE-NOTES.md
Comment (canyon289) [reply RELEASE-NOTES.md:28]: Made the changes

**Comment by michaelosthege** [comment]
> 
> 
> This is just a question about booking keeping. Should we merge anything other than the release notes and a version bump in a release pr? The code will end up being the same just curious about the git history type stuff
> 
> My opinion is no, if anyone else agrees Ill split up this PR

I'm fine with having this in the same PR, but the git history is too messy to be merged without squashing.
If you squash the git history into two commits about readme and docs respectively (then force push the PR branch) we could just rebase-merge.

**Comment by canyon289** [comment]
Struggling with git at the moment so just ignore this history for a bit. Here's what I have locally and not sure why its not being reflected here

image

**Comment by canyon289** [comment]
Closing this in favor of https://github.com/pymc-devs/pymc3/pull/4962. No idea why sayams commit is not showing up here

**Pull request closed without merging**
